js/main.js

```diff
@@ -9,21 +9,56 @@ function photoUpload(event) {
 }
 
 var elEntryForm = document.querySelector('#journal-form');
+
 elEntryForm.addEventListener('submit', formSubmit);
 
 function formSubmit(event) {
   event.preventDefault();
-  var newObj = {};
-  newObj.title = elEntryForm.elements.title.value;
-  newObj.url = elEntryForm.elements.url.value;
-  newObj.notes = elEntryForm.elements.notes.value;
-  newObj.entryId = data.nextEntryId++;
-  data.entries.unshift(newObj);
-  elImgURL.setAttribute('src', '/images/placeholder-image-square.jpg');
-  switchView('entries');
-  elEntryForm.reset();
-  var newEntryDom = singleEntry(newObj);
-  listUl.prepend(newEntryDom);
+
+  var newObj = {
+    title: elEntryForm.elements.title.value,
+    url: elEntryForm.elements.url.value,
+    notes: elEntryForm.elements.notes.value,
+    entryId: data.nextEntryId++
```

Comment:
You shouldn't do this unless you have confirmed that this is a new entry. Otherwise, whenever a user edits an entry, a new id will be used and immediately discarded.